h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -140,11 +141,19 @@ private Frame prepareTrainingLevelOneFrame(StackedEnsembleModel.StackedEnsembleP
           throw new H2OIllegalArgumentException("Failed to find the xval predictions frame id. . .  Looks like keep_cross_validation_predictions wasn't set when building the models.");
 
         Frame aFrame = DKV.getGet(aModel._output._cross_validation_holdout_predictions_frame_id);
+
         if (null == aFrame)
           throw new H2OIllegalArgumentException("Failed to find the xval predictions frame. . .  Looks like keep_cross_validation_predictions wasn't set when building the models, or the frame was deleted.");
 
         baseModels.add(aModel);
-        baseModelPredictions.add(aFrame);
+        if(!aModel._output.isMultinomialClassifier()){
+            baseModelPredictions.add(aFrame);
+        }else {
+            List<String> predColNames= new ArrayList<>(Arrays.asList(aFrame.names()));
+            predColNames.remove("predict");
+            String[] multClassNames  = predColNames.toArray(new String[0]);
```

Comment:
The column removal can be done in a single loop without no need to create list -> remove col -> and create a new array